blocks/contacts/contacts.css

```diff
@@ -0,0 +1,9 @@
+.contacts{
```

Comment:
The last thing for now, we need to set paddings for your section to match the design (both top and bottom). I know the design is a bit  wacky on that 🤪, but that is really contact's section top padding
<img width="903" alt="Screenshot 2023-11-16 at 6 09 37 PM" src="https://github.com/p-foundation/pink-app/assets/76534205/c7d0b9d7-fc90-4184-bff0-00b010b37fa6">

Reply:
@naumch1k I added `padding-top` and `padding-bottom` to screen size 1200px.However I added only `padding-top` to screen sizes 660px and 320px

Reply:
@SergeyAlexandrowich Thank you! It's alright, you, surely, do not have to add padding-bottom to 660px and 320px screen sizes if it doesn't change =)